packages/core/src/ceramic.ts

```diff
@@ -535,11 +547,10 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
   }
 
   async _publishVersionMetrics() {
-    const ipfsVersion = (await this.ipfs.id()).agentVersion
     Metrics.observe(VERSION_INFO, 1, {
       jsCeramicVersion: this._versionInfo.cliPackageVersion,
       jsCeramicGitHash: this._versionInfo.gitHash,
-      ceramicOneVersion: ipfsVersion,
+      ceramicOneVersion: this._versionInfo.ceramicOneVersion,
```

Comment:
a small downside of this is that if ipfs/ceramic-one is upgraded out from under a running js-ceramic node, the js-ceramic node will continue to report the old version of ipfs/C1 until js-ceramic restarts.  Since it should be rare to upgrade C1 without also at least restarting js-ceramic, I think this is acceptable.

Reply:
yeah i think thats a really weird edge case, and we can also tell when it happens.  i think this metric is fine